### DISCUSSION TITLE
Support ENS160 + AHT21

### DISCUSSION THREAD

**Discussion by Wheemer** [ESPresense/ESPresense#843] [Ideas]
Hello;

Was wondering if it would be possible to add support for these combo sensors with air quality/gas detection?

[https://www.ebay.ca/itm/144731743011?hash=item21b2af2323:g:cjcAAOSw8OtjLYPq&amdata=enc%3AAQAHAAAA4P7JEEyD5UnFp6nY1PU2pfEjs0WX5RA5oCrYoBSm7sFLTV8aDlSU%2B4sGQSnWn4U%2FDyWu3Nuagogck%2FQPLGjOdFQQDL%2FMvQxiEKoREvCsbjIn2dhBaB2hhHKaaoTRk6HcuQjhjhTDUqeckDe4CqczCtgNG6eQOqfrzIE44mPUkUQnFBETXlgqKj1VDsVCyAyBKObzfZJJLO5miKlAsyy%2BSBJJYv2OLCFK9loMF7ogijOYk195n1VhK2OpyfagSvw70xJPFskPb4zWFFbjt9hfJFv6aSNCMyq7ONNjMU0YSnIk%7Ctkp%3ABk9SR6jVju7ZYQ](url)

**Comment by Wheemer** [comment]
At least support for the ENS160 would be appreciated, I see that the AHT21 part would work fine.

**Reply by Wheemer** [reply]
In order to get the ens160 working in ESPhome this was the additions I needed to make:

`external_components:
  - source: github://pr#4243
    components: ens160

sensor:
  - platform: ens160
    id: ens160_sensor
    address: 0x53
    eco2:
      name: eCO2
      id: eco2
    tvoc:
      name: TVOC
      id: tvoc
    aqi:
      name: AQI
      id: aqi
    update_interval: 30s`

I'm not sure if this helps, but can't hurt.

**Comment by Geosham233** [comment]
I have a problem when using these sensors combined with the ESP8266. Has anyone used these components?

**Comment by onlize** [comment]
Does anyone has a schematics for this device? What is the difference between the VIN and 3V3 pins? Which phone should I use to power this board and how to connect it to WeMos Mini d!?
